test/test_nn.py

```diff
@@ -12871,6 +12871,22 @@ def cosine_distance(x, y):
             self.assertEqual(functional, modular, atol=1e-6, rtol=1e-6)
             self.assertEqual(traced, modular, atol=1e-6, rtol=1e-6)
 
+    def test_to_complex(self, device):
+        m = nn.Linear(3, 5).to(device)
+        self.assertIs(m, m.to(device))
+        m.to(torch.complex64)
```

Comment:
nit - we can also just write: 
`m.to(torch.cfloat)`
...
`m.to(torch.cdouble)` (since we have been more commonly using the alias name in other tests as well ...

Reply:
Okay, I can change that. I was using `complex64`/`complex128` because that is what is used in tensor's `__repr__`, when tensors are printed.